### PR TITLE
Lesson#2

### PR DESCRIPTION
**Pull request by AlexPFrank**
Lesson#2 completed

### CODE REVIEW

File: Lesson#2/Task2.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="UTF-8">
+	<title>Lesson#2 Task4</title>
```
Comment (timchukd) [inline Lesson#2/Task2.html]: task2 should be in the title

File: Lesson#2/Task3.html [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+<body>
+	<h1 class="align-center">Task 3</h1>
+	<hr/>
+	<form>
+		Name
+		<br/>
+		<input type="text" name="firstname"/>
+		<br/>
+		<br/>
```
Comment (timchukd) [inline Lesson#2/Task3.html]: second tag <br/> is not needed here
if you want to add more space, you should do it with styles to add margin-bottom for example

p.s. please check rest of the code

**Review by timchukd — CHANGES_REQUESTED** [review]
looks nice 👍 

please fix few findings, please delete task1 - 4 from root directory and good to go ;)

File: Lesson#2/Task2.html (hunk 1 above)
Comment (AlexPFrank) [reply Lesson#2/Task2.html]: Fixed

File: Lesson#2/Task3.html (hunk 2 above)
Comment (AlexPFrank) [reply Lesson#2/Task3.html]: Fixed

**Review by timchukd — APPROVED** [review]

**Comment by timchukd** [comment]
merge it

**Pull request merged**